File: python/1000-1099/1040-1049/1040.py
n1, n2, n3, n4 = input().split()
media = (float(n1) * 2 + float(n2) * 3 + float(n3) * 4 + float(n4)) / 10
print(f"Media: {media:.1f}")
if media >= 7:
    print("Aluno aprovado.")
elif media >= 5 and media < 7:
    print("Aluno em exame.")
    exam = float(input())
    print(f"Nota do exame: {exam:.1f}")
    media = (media + exam) / 2
    if media >= 5:
        print("Aluno aprovado.")
    else:
        print("Aluno reprovado.")
    print(f"Media final: {media:.1f}")
else:
    print("Aluno reprovado.")

File: C99/1000-1099/1040-1049/1040.c
#include <stdio.h>

int main()
{
    double n1, n2, n3, n4, media, ex;
    scanf("%lf %lf %lf %lf", &n1, &n2, &n3, &n4);
    n1 *= 0.2;
    n2 *= 0.3;
    n3 *= 0.4;
    n4 *= 0.1;
    media = n1 + n2 + n3 + n4;
    printf("Media: %.1lf\n", media);
    if (media >= 7)
    {
        printf("Aluno aprovado.\n");
    }
    else if (media >= 5 && media < 7)
    {
        printf("Aluno em exame.\n");
        scanf("%lf", &ex);
        media = (media + ex) / 2;
        printf("Nota do exame: %.1lf\n", ex);
        if (media >= 5)
        {
            printf("Aluno aprovado.\n");
        }
        else
        {
            printf("Aluno reprovado.\n");
        }
        printf("Media final: %.1lf\n", media);
    }
    else if (media < 5)
    {
        printf("Aluno reprovado.\n");
    }
    return 0;
}